charts/mongodb/values.yaml

```diff
@@ -7,9 +7,6 @@ database:
   password: pAssw0rd
```

Comment:
Are we using these values? in the deployment config, I see we only refer to the secrets, also can we empty these values and a comment to configure it?